src/components/Ledger/LedgerForm/LedgerForm.js

```diff
@@ -0,0 +1,157 @@
+import { Field } from 'redux-form';
+
+import stripesForm from '@folio/stripes/form';
+import {
+  Button,
+  Col,
+  Pane,
+  PaneMenu,
+  Paneset,
+  Row,
+  TextArea,
+  TextField,
+} from '@folio/stripes/components';
+import { ViewMetaData } from '@folio/stripes/smart-components';
+import {
+  AcqUnitsField,
+  FieldSelection,
+  validateRequired,
+} from '@folio/stripes-acq-components';
+
+import {
+  CREATE_UNITS_PERM,
+  MANAGE_UNITS_PERM,
+} from '../../../common/const';
+import {
+  LEDGER_FORM,
+  LEDGER_STATUS_OPTIONS,
+} from '../constants';
+
+const CREATE_LEDGER_TITLE = <FormattedMessage id="ui-finance.ledger.form.title.create" />;
+const EDIT_LEDGER_TITLE = <FormattedMessage id="ui-finance.ledger.form.title.edit" />;
+
+const getLastMenu = (handleSubmit, pristine, submitting) => {
+  return (
+    <PaneMenu>
+      <Button
+        data-test-button-save-ledger
+        marginBottom0
+        buttonStyle="primary"
+        onClick={handleSubmit}
+        type="submit"
+        disabled={pristine || submitting}
+      >
+        <FormattedMessage id="ui-finance.save" />
+      </Button>
+    </PaneMenu>
+  );
+};
+
+const LedgerForm = ({
+  onCancel,
+  initialValues,
+  handleSubmit,
+  pristine,
+  submitting,
+}) => {
+  const isEditMode = Boolean(initialValues.id);
+  const lastMenu = getLastMenu(handleSubmit, pristine, submitting);
+  const metadata = initialValues.metadata;
+
+  return (
+    <form>
+      <Paneset>
+        <Pane
+          defaultWidth="fill"
+          dismissible
+          id="pane-ledger-form"
+          onClose={onCancel}
+          lastMenu={lastMenu}
+          paneTitle={isEditMode ? EDIT_LEDGER_TITLE : CREATE_LEDGER_TITLE}
+        >
+          <Row>
+            <Col
+              xs={12}
+              md={8}
+              mdOffset={2}
+            >
+              {metadata && <ViewMetaData metadata={metadata} />}
+              <Row>
+                <Col data-test-col-ledger-form-name xs={3}>
+                  <Field
+                    component={TextField}
+                    label={<FormattedMessage id="ui-finance.ledger.name" />}
+                    name="name"
+                    type="text"
+                    required
+                    validate={validateRequired}
+                  />
+                </Col>
+
+                <Col data-test-col-ledger-form-code xs={3}>
+                  <Field
+                    component={TextField}
+                    label={<FormattedMessage id="ui-finance.ledger.code" />}
+                    name="code"
+                    type="text"
+                    required
+                    validate={validateRequired}
+                  />
+                </Col>
+
+                <Col data-test-col-ledger-form-status xs={3}>
+                  <FieldSelection
+                    dataOptions={LEDGER_STATUS_OPTIONS}
+                    id="ledger-status"
+                    label={<FormattedMessage id="ui-finance.ledger.status" />}
```

Comment:
it should be `labelId` to avoid console warning
```suggestion
                    labelId="ui-finance.ledger.status"
```